cluster-autoscaler/cloudprovider/aws/aws-sdk-go/aws/awserr/error.go

```diff
@@ -10,24 +10,23 @@ package awserr
 //
 // Example:
 //
-//     output, err := s3manage.Upload(svc, input, opts)
-//     if err != nil {
-//         if awsErr, ok := err.(awserr.Error); ok {
-//             // Get error details
-//             log.Println("Error:", awsErr.Code(), awsErr.Message())
-//
-//             // Prints out full error message, including original error if there was one.
-//             log.Println("Error:", awsErr.Error())
-//
-//             // Get original error
-//             if origErr := awsErr.OrigErr(); origErr != nil {
-//                 // operate on original error.
-//             }
-//         } else {
-//             fmt.Println(err.Error())
-//         }
-//     }
-//
+//	output, err := s3manage.Upload(svc, input, opts)
```

Comment:
Is this (and all the following) changes to `aws-sdk-go` necessary? Isn't this autogenerated/vendored manually?

Reply:
I was able to remove these changes without triggering the gofmt.